cpp/oneapi/dal/algo/rbf_kernel/backend/gpu/compute_kernel_dense_dpc.cpp

```diff
@@ -110,6 +110,7 @@ void compute_rbf(sycl::queue& queue,
     {
         ONEDAL_PROFILER_TASK(rbf_kernel.gemm, queue);
         gemm_event = pr::gemm(queue, x_nd, y_nd.t(), res_nd, alpha, beta, { reduce_y_event });
+        gemm_event.wait_and_throw();
```

Comment:
```suggestion
        // TODO: Review this call later and decide if it is necessary 
        gemm_event.wait_and_throw();
```

Please mark it as a thing to change